Content.Server/GameObjects/Components/Power/ApcNetComponents/PowerReceiverUsers/LightBulbComponent.cs

```diff
@@ -52,7 +53,7 @@ public class LightBulbComponent : Component, ILand, IBreakAct
             set
             {
                 _color = value;
-                OnLightColorChange?.Invoke(this, null);
+                OnLightColorChange?.Invoke(this, new EventArgs());
```

Comment:
Does this work if you do EventHandle<EventArgs?>?

Reply:
worked